app/src/db.js

```diff
@@ -59,13 +58,18 @@ function disconnect() {
  */
 function addUser(userId, accessToken) {
```

Comment:
What if the user already exists? Need to check for it and if it does, update its accesstoken instead.